lib/std/compress/deflate.zig

```diff
@@ -125,6 +127,9 @@ const Huffman = struct {
 
         self.last_code = codes[PREFIX_LUT_BITS + 1];
         self.last_index = offset[PREFIX_LUT_BITS + 1] - self.count[PREFIX_LUT_BITS + 1];
+
+        if (left > 0)
```

Comment:
Minor nit, why not move the check below the loop that computes `left`?

Reply:
I didn't want to potentially leave `self.last_code`/`self.last_index` uninitialized since `error.IncompleteSet` is not always treated as an error by the caller, and if `decode` uses them when uninitialized it can lead to index-out-of-bounds/UB.